README.md

```diff
@@ -95,14 +100,17 @@ Upon success, you should see a dashboard similar to:
 Modify your application to periodically send the request to Upgrade Responder server every `requestIntervalInMinutes` where `requestIntervalInMinutes` is the value returned from Upgrade Responder sever when your application makes an upgrade request to it. 
 Your application should send one `POST` request every `requestIntervalInMinutes`. 
 The request's body should be in the format:
-   ```json
-   {
+```json
+{
     "appVersion": "v0.8.1",
-    "extraInfo": {
-        "fieldKey": "value"
+    "extraTagInfo": {
```

Comment:
It will be good to explain what's difference between `extraTagInfo` and `extraFieldInfo`, so the user will know when to use which.